From 3d9ced9e340678cc02b1a36c2139492c95ef15a6 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Wed, 12 Aug 2020 12:46:42 -0400
Subject: [PATCH 2/2] Issue 50933 - Fix OID change between 10rfc2307 and
 10rfc2307compat

Bug Description:  10rfc2307compat changed the OID for nisMap objectclass to
                  match the standard OID, but this breaks replication with
                  older versions of DS.

Fix Description:  Continue to use the old(invalid?) oid for nisMap so that
                  replication does not break in a mixed version environment.

Fixes: https://pagure.io/389-ds-base/issue/50933

Reviewed by: firstyear & tbordaz(Thanks!!)
---
 ldap/schema/10rfc2307compat.ldif | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/schema/10rfc2307compat.ldif b/ldap/schema/10rfc2307compat.ldif
index 78c588d08..8ba72e1e3 100644
--- a/ldap/schema/10rfc2307compat.ldif
+++ b/ldap/schema/10rfc2307compat.ldif
@@ -253,7 +253,7 @@ objectClasses: (
   MAY ( nisNetgroupTriple $ memberNisNetgroup $ description )
   )
 objectClasses: (
-  1.3.6.1.1.1.2.9 NAME 'nisMap' SUP top STRUCTURAL
+  1.3.6.1.1.1.2.13 NAME 'nisMap' SUP top STRUCTURAL
   DESC 'A generic abstraction of a NIS map'
   MUST nisMapName
   MAY description
-- 
2.26.2

